packages/network/test/unit/agent_spec.ts

```diff
@@ -148,6 +204,15 @@ describe('lib/agent', function () {
                 url: `localhost:${HTTPS_PORT}`,
               })
             }
+
+            if (this.clientCert) {
+              // If a client cert has been assigned to a TLS connection, the key for the TLSSocket
+              // will include the public certificate
+              const socketKey = Object.keys(this.agent.httpsAgent.sockets).filter((key) => key.includes(`localhost:${HTTPS_PORT}`))
+
+              expect(socketKey.length).to.eq(1, 'There should only be a single localhost TLS Socket')
+              expect(socketKey[0]).to.contain(this.clientCert, 'A client cert should be been used for the TLS Socket')
+            }
```

Comment:
ditto with earlier, let's make a new test for this instead of modifying the HTTPS tests that exist